src/openrct2-ui/windows/Guest.cpp

```diff
@@ -1973,7 +1974,12 @@ static rct_string_id window_guest_inventory_format_item(Peep* peep, int32_t item
         case SHOP_ITEM_PHOTO:
             ride = get_ride(peep->Photo1RideRef);
             if (ride != nullptr)
-                ride->FormatNameTo(gCommonFormatArgs + 6);
+            {
+                ft = Formatter::Common();
+                ft.Increment(6);
```

Comment:
You should rewind followed by the increment to show is keeping some values. Will help in the future